vitrina/users/models.py

```diff
@@ -83,6 +84,15 @@ def is_coordinator(self):
     def is_supervisor(self):
         return bool(self.representative_set.filter(role=Representative.SUPERVISOR))
 
+    @property
+    def is_gov_organization_manager(self) -> bool:
+        organization_representative = Representative.objects.filter(
+            user=self, content_type=ContentType.objects.get_for_model(Organization), object_id=self.organization_id
```

Comment:
Even if `content_type` & `object_id` is used to correctly identify which organization this entity (Representative) represents, I'd still suggest adding a filter for the `Representative.organization` as well since it is still a field that is not deprecated.

Unless there is some part that I am missing here or did not read from the task description, let's discuss then!